From 9abe8151ed6e38caf0928890fee30157d3c51e5d Mon Sep 17 00:00:00 2001
From: Lu Hui <luhui@sipeed.com>
Date: Sun, 7 Apr 2024 09:41:20 +0800
Subject: [PATCH] update

---
 sample/common/sample_common_sensor.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sample/common/sample_common_sensor.c b/sample/common/sample_common_sensor.c
index 120ff09..4cff919 100644
--- a/sample/common/sample_common_sensor.c
+++ b/sample/common/sample_common_sensor.c
@@ -745,6 +745,9 @@ CVI_S32 SAMPLE_COMM_SNS_GetDevAttr(SAMPLE_SNS_TYPE_E enSnsType, VI_DEV_ATTR_S *p
 	case GCORE_GC1084_SLAVE2_MIPI_1M_30FPS_10BIT:
 	case GCORE_GC4653_MIPI_4M_30FPS_10BIT:
 	case GCORE_GC4653_SLAVE_MIPI_4M_30FPS_10BIT:
+		// omnivision
+	case OV_OS04A10_MIPI_4M_1440P_30FPS_12BIT:
+	case OV_OS04A10_MIPI_4M_1440P_30FPS_10BIT_WDR2TO1:
 	case TECHPOINT_TP2850_MIPI_2M_30FPS_8BIT:
 	case TECHPOINT_TP2850_MIPI_4M_30FPS_8BIT:
 		pstViDevAttr->enBayerFormat = BAYER_FORMAT_GR;
@@ -1032,6 +1035,9 @@ CVI_S32 SAMPLE_COMM_SNS_GetIspAttrBySns(SAMPLE_SNS_TYPE_E enSnsType, ISP_PUB_ATT
 	case GCORE_GC1084_SLAVE2_MIPI_1M_30FPS_10BIT:
 	case GCORE_GC4653_MIPI_4M_30FPS_10BIT:
 	case GCORE_GC4653_SLAVE_MIPI_4M_30FPS_10BIT:
+		// omnivision
+	case OV_OS04A10_MIPI_4M_1440P_30FPS_12BIT:
+	case OV_OS04A10_MIPI_4M_1440P_30FPS_10BIT_WDR2TO1:
 	case TECHPOINT_TP2850_MIPI_2M_30FPS_8BIT:
 	case TECHPOINT_TP2850_MIPI_4M_30FPS_8BIT:
 		// on licheervnano
-- 
2.34.1

