From 72e2cbd18ca629e7d04070edf33c54e735964e5b Mon Sep 17 00:00:00 2001
From: scpcom <scpcom@gmx.de>
Date: Fri, 25 Oct 2024 18:34:03 +0200
Subject: [PATCH] re-add CVI_VDEC_GetFd/CVI_VDEC_CloseFd to cvi_vdec.h

---
 include/cvi_vdec.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/include/cvi_vdec.h b/include/cvi_vdec.h
index 6ccdba0..bc3989a 100644
--- a/include/cvi_vdec.h
+++ b/include/cvi_vdec.h
@@ -73,6 +73,20 @@ CVI_S32 CVI_VDEC_StopRecvStream(VDEC_CHN VdChn);
  */
 CVI_S32 CVI_VDEC_QueryStatus(VDEC_CHN VdChn, VDEC_CHN_STATUS_S *pstStatus);
 
+/* Get File Descriptor
+ *
+ * @param VdChn(In): channel number
+ * @return Error code (0 if successful)
+ */
+CVI_S32 CVI_VDEC_GetFd(VDEC_CHN VdChn);
+
+/* Close File Descriptor
+ *
+ * @param VdChn(In): channel number
+ * @return Error code (0 if successful)
+ */
+CVI_S32 CVI_VDEC_CloseFd(VDEC_CHN VdChn);
+
 /* Reset Channel
  *
  * @param VdChn(In): channel number
-- 
2.34.1

