diff --git a/sensor/cv182x/sms_sc035hgs/sc035hgs_cmos_param.h b/sensor/cv182x/sms_sc035hgs/sc035hgs_cmos_param.h
index 4cd93a9d66e8a8fe87d39e0e1d1c1fbc830e1a95..48dfc31b387342a7abfacd416f38fd4da587f30b 100644
--- a/sensor/cv182x/sms_sc035hgs/sc035hgs_cmos_param.h
+++ b/sensor/cv182x/sms_sc035hgs/sc035hgs_cmos_param.h
@@ -99,9 +99,9 @@ struct combo_dev_attr_s sc035hgs_rx_attr = {
 	.mac_clk = RX_MAC_CLK_200M,
 	.mipi_attr = {
 		.raw_data_type = RAW_DATA_12BIT,
-		.lane_id = {2, 0, 1, -1, -1},
+		.lane_id = {2, 4, 3, -1, -1},
 		.wdr_mode = CVI_MIPI_WDR_MODE_NONE,
-		.pn_swap = {1, 1, 1, 0, 0},
+		.pn_swap = {0, 0, 0, 0, 0},
 	},
 	.mclk = {
 		.cam = 0,
